sound/soc/sof/ops.h

```diff
@@ -488,6 +488,16 @@ static inline int snd_sof_pcm_platform_ack(struct snd_sof_dev *sdev,
 	return 0;
 }
 
+static inline int snd_sof_pcm_get_stream_position(struct snd_sof_dev *sdev,
```

Comment:
the git commit has still "update_stream_position" but code as get_stream_position

Reply:
sure, thanks